src/Visualization/Python/OpenVolfiles.py

```diff
@@ -61,130 +74,151 @@ def open_volfiles_command(f):
-        "h5_files",
-        nargs=-1,
-        type=click.Path(
-            exists=True, file_okay=True, dir_okay=False, readable=True
-        ),
-    )
-    @click.option(
-        "--subfile-name",
-        "-d",
-        help="Name of subfile within h5 file containing volume data to plot.",
-    )
-    @click.option(
-        "--list-vars",
-        "-l",
-        is_flag=True,
-        help="Print available variables and exit.",
-    )
-    @click.option(
-        "--var",
-        "-y",
-        "vars_patterns",
-        multiple=True,
-        help=(
-            "Variables to plot. List any tensor components "
-            "in the volume data file, such as 'Shift_x'. "
-            "Also accepts glob patterns like 'Shift_*'."
-        ),
-    )
-    @click.option(
-        "--step",
-        callback=parse_step,
-        help=(
-            "Observation step number. Specify '-1' or 'last' "
-            "for the last step in the file. "
-            "Mutually exclusive with '--time'."
-        ),
-    )
-    @click.option(
-        "--time",
-        type=float,
-        help=(
-            "Observation time. "
-            "The observation step closest to the specified "
-            "time is selected. "
-            "Mutually exclusive with '--step'."
-        ),
-    )
-    @functools.wraps(f)
-    def wrapper(
-        h5_files,
-        subfile_name,
-        list_vars,
-        vars_patterns,
-        step,
-        time,
-        **kwargs,
-    ):
-        # Script should be a noop if input files are empty
-        if not h5_files:
-            return
-
-        # Print available subfile names and exit
-        if not subfile_name:
-            import rich.columns
-
-            with spectre_h5.H5File(h5_files[0], "r") as open_h5_file:
-                available_subfiles = open_h5_file.all_vol_files()
-            if len(available_subfiles) == 1:
-                subfile_name = available_subfiles[0]
-            else:
-                rich.print(rich.columns.Columns(available_subfiles))
+    def decorator(f):
+        @click.argument(
+            "h5_files",
+            nargs=-1,
+            type=click.Path(
+                exists=True, file_okay=True, dir_okay=False, readable=True
+            ),
+        )
+        @click.option(
+            "--subfile-name",
+            "-d",
+            help=(
+                "Name of subfile within h5 file containing volume data to plot."
+            ),
+        )
+        @click.option(
+            "--list-vars",
+            "-l",
+            is_flag=True,
+            help="Print available variables and exit.",
+        )
+        @click.option(
+            "--var",
+            "-y",
+            "vars_patterns",
+            multiple=multiple_vars,
+            help=(
+                "Variable to plot. List any tensor components "
+                "in the volume data file, such as 'Shift_x'. "
+                "Also accepts glob patterns like 'Shift_*'."
```

Comment:
Will the glob work if you don't allow multiple vars?

Reply:
The glob works, but you'll get an error if it matches more than 1 variable. That's useful behavior I think (don't have to type the full name).